pkg/backup/util/util.go

```diff
@@ -326,16 +334,22 @@ func GetBackupDataPath(provider v1alpha1.StorageProvider) (string, string, error
 	return fmt.Sprintf("%s://%s", string(storageType), backupPath), "", nil
 }
 
-func ValidateBackup(backup *v1alpha1.Backup) error {
+func ValidateBackup(backup *v1alpha1.Backup, tikvImage string) error {
 	ns := backup.Namespace
 	name := backup.Name
 
-	if backup.Spec.From.Host == "" {
-		return fmt.Errorf("missing cluster config in spec of %s/%s", ns, name)
-	}
+	if !canSkipSetGCLifeTime(tikvImage) {
+		if backup.Spec.From == nil {
+			return fmt.Errorf("missing cluster config in spec of %s/%s", ns, name)
+		} else {
+			if backup.Spec.From.Host == "" {
+				return fmt.Errorf("missing cluster config in spec of %s/%s", ns, name)
+			}
 
-	if backup.Spec.From.SecretName == "" {
-		return fmt.Errorf("missing tidbSecretName config in spec of %s/%s", ns, name)
+			if backup.Spec.From.SecretName == "" {
+				return fmt.Errorf("missing tidbSecretName config in spec of %s/%s", ns, name)
+			}
+		}
```

Comment:
We can define this block as validateAccessConfig, and then:
if backup.Spec.BR == nil, we should call this function;
else if !canSkipSetGCLifeTime(tikvImage), call this.